reporter_api/models/user_model.py

```diff
@@ -30,4 +32,9 @@ def create_reporter(self,args):
     def get_reporter(self,user_id):
         for reporter in self.users:
             if reporter['user_id'] == user_id:
-                return reporter
```

Comment:
To avoid repetition on this say:
def get_reporter(self,user_id):
        for reporter in self.users:
            if reporter['user_id'] == user_id:
            return reporter 
return at that point for proper conitution of the for loop